container_linux.go

```diff
@@ -125,7 +126,17 @@ func (c *linuxContainer) commandTemplate(p *Process, childPipe *os.File) (*exec.
 		Path: c.initPath,
 		Args: c.initArgs,
 	}
-	cmd.Stdin = p.Stdin
+	if p.Stdin != nil {
+		stdin, err := cmd.StdinPipe()
+		if err != nil {
+			return nil, err
+		}
+		go func() {
+			io.Copy(stdin, p.Stdin)
+			stdin.Close()
+		}()
+	}
+
```

Comment:
@LK4D4 In many cases it works fine without this hack, so I would prefer to use it only if it's required. For example if I set a terminal as Stdin, I expect that a process will be executed with the terminal and it will be able to call tty ioctls on the stdin descriptor. This patch breaks this ability.